packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -374,6 +380,18 @@ class SemanticsProperties extends DiagnosticableTree {
   /// Defaults to the ambient [Directionality].
   final TextDirection textDirection;
 
+  /// Provides a traversal sorting order for this [Semantics] node.
+  ///
+  /// This is used to describe the order in which the semantic node should be
+  /// traversed by the accessibility services on the platform (e.g. VoiceOver
+  /// on iOS and TalkBack on Android).
+  ///
+  /// If [sortOrder.discardParentOrder] is false (the default), [sortOrder]'s
```

Comment:
I find this paragraph hard to understand. Maybe add an example?

Reply:
Or add an "See also" entry linking to SemanticsSortOrder, which seems to explain this in more detail.

Reply:
Added the See also.